data_structures/binary_tree/avl_tree.py

```diff
@@ -256,18 +260,18 @@ class AVLtree:
     *************************************
     """
 
-    def __init__(self):
-        self.root = None
+    def __init__(self) -> None:
+        self.root: my_node = None
 
     def get_height(self):
         #        print("yyy")
         return get_height(self.root)
 
-    def insert(self, data):
+    def insert(self, data: int) -> None:
         print("insert:" + str(data))
         self.root = insert_node(self.root, data)
 
-    def del_node(self, data):
+    def del_node(self, data: int) -> None:
         print("delete:" + str(data))
         if self.root is None:
             print("Tree is empty!")
```

Comment:
Algorithmic functions/methods should not print() as discussed in CONTRIBUTING.md.  Please raise an appropriate exception instead.